### PR TITLE
Axios

### PR DESCRIPTION
**Pull request by freezeboy**
Add Axios support to servant-js

When urlPrefix is merged #172, we can uncomment line 80 in servant-js/src/Servant/JS/Axios.hs

### CODE REVIEW

File: servant-js/test/Servant/JSSpec.hs [hunk 1]
```diff
@@ -71,9 +74,35 @@ spec = describe "Servant.JQuery" $ do
     generateJSSpec JQueryCustom  (JQ.generateJQueryJSWith customOptions)
     generateJSSpec Angular       (NG.generateAngularJS NG.defAngularOptions)
     generateJSSpec AngularCustom (NG.generateAngularJSWith NG.defAngularOptions customOptions)
+    generateJSSpec Axios        (AX.generateAxiosJS AX.defAxiosOptions)
+    generateJSSpec AxiosCustom  (AX.generateAxiosJSWith (AX.defAxiosOptions { withCredentials = True }) customOptions)
     
     angularSpec    Angular
-    angularSpec    AngularCustom
+    axiosSpec
+    --angularSpec    AngularCustom
```
Comment (alpmestan) [inline servant-js/test/Servant/JSSpec.hs]: why is `angularSpec AngularCustom` commented out now?

**Comment by freezeboy** [comment]
The spec wasn't really customised, so the checks were exactly the same..

**Comment by alpmestan** [comment]
Oh ok. Ready to merge?

**Comment by freezeboy** [comment]
I think so

**Pull request merged**